{
	"cve": {
		"data_type": "CVE",
		"data_format": "MITRE",
		"data_version": "4.0",
		"CVE_data_meta": {
			"ID": "CVE-2020-9306",
			"ASSIGNER": "cve@mitre.org"
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": []
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"url": "https://github.com/fireeye/Vulnerability-Disclosures/blob/master/FEYE-2020-0019/FEYE-2020-0019.md",
					"name": "https://github.com/fireeye/Vulnerability-Disclosures/blob/master/FEYE-2020-0019/FEYE-2020-0019.md",
					"refsource": "CONFIRM",
					"tags": []
				},
				{
					"url": "https://www.fireeye.com/blog/threat-research.html",
					"name": "https://www.fireeye.com/blog/threat-research.html",
					"refsource": "MISC",
					"tags": []
				},
				{
					"url": "https://www.fireeye.com/blog/threat-research/2021/02/solarcity-exploitation-of-x2e-iot-device-part-one.html",
					"name": "https://www.fireeye.com/blog/threat-research/2021/02/solarcity-exploitation-of-x2e-iot-device-part-one.html",
					"refsource": "MISC",
					"tags": []
				},
				{
					"url": "https://www.fireeye.com/blog/threat-research/2021/02/solarcity-exploitation-of-x2e-iot-device-part-two.html",
					"name": "https://www.fireeye.com/blog/threat-research/2021/02/solarcity-exploitation-of-x2e-iot-device-part-two.html",
					"refsource": "MISC",
					"tags": []
				}
			]
		},
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "Tesla SolarCity Solar Monitoring Gateway through 5.46.43 has a \"Use of Hard-coded Credentials\" issue because Digi ConnectPort X2e uses a .pyc file to store the cleartext password for the python user account."
				}
			]
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": []
	},
	"impact": {},
	"publishedDate": "2021-02-18T00:15Z",
	"lastModifiedDate": "2021-02-18T00:15Z"
}